tests/php/Core/EnvironmentTest.php

```diff
@@ -79,7 +79,6 @@ public function provideHasEnv()
         $valueOptions = [
             true,
             false,
-            null,
```

Comment:
I've removed this as it's not compatible with the `isset()` approach where a key set with a value of `null` will return `false`

Setting `null` values for environment, $_SERVER or $_ENV seems very non-sensical.  Also in the case of the `putenv` it's converted to `"null"` via `json_encode()`, which isn't useful in the context this code was originally developed for to assist [Deprecation::varAsBoolean()](https://github.com/silverstripe/silverstripe-framework/blob/4/src/Dev/Deprecation.php#L455)